### DISCUSSION TITLE
Feature Request: Intuitive site searching

### DISCUSSION THREAD

**Discussion by georg-stone** [zen-browser/desktop#2840] [Ideas]
Microsoft edge has this cool feature where you can type a website to search (like amazon.com), press space, and then type your query. It automatically adds search engines to your browser when you go to new websites. I think it is very intuitive and could be added to Zen Browser.

**Comment by ghost** [comment]
Is the feature request that websites are added automatically? You can already add them manually in Settings > Search Shortcuts.

**Reply by georg-stone** [reply]
Yeah, and the ability to intuitively search your added search engines based on what I described.

**Comment by s3r3n3ty** [comment]
The important part here is the way a site search is initiated and right now it sucks.
Currently to perform a site search before i type anything I have to activate site search by pressing @ (not the most convenient to press) followed by the site and then my query.

Since Zens meta is pretty much just to follow Arc the obvious answer is start typing the site you want to search, press tab as soon as it shows up, enter query.

So instead of
cmd+t, shift+2(@), [name of site], enter, [query];
as it is right now, you'd just: 
cmd+t, [name of site], tab, [query];

one less keypress and more intuitive flow, especially to arc users switching to zen

This is literally about changing keybinds since everything else is already there.

A cherry on top would be to automatically recognize and add open search format sites the user visits.
ex. I go to ebay and search something. Zen automatically recognizes the open search format url and adds it to site search so next time i can search ebay directly from the new tab bar, without having to open ebay first.
